### PR TITLE
Places fixes

### PR DESCRIPTION
**Pull request by namArora3112**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainApplication.kt [hunk 1]
```diff
@@ -74,7 +74,7 @@ class MainApplication : Application(), ReactApplication {
     super.onCreate()
     MobileCore.setApplication(this)
     MobileCore.setLogLevel(LoggingMode.DEBUG)
-    MobileCore.configureWithAppID("example")
+    MobileCore.configureWithAppID("94f571f308d5/cbd0aa0847bc/launch-10aaa7c74b4e-development")
```
Comment (yangyansong-adbe) [inline apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainApplication.kt:77]: Let's remove this app id.

File: packages/places/android/src/main/java/com/adobe/marketing/mobile/reactnative/places/RCTAEPPlacesModule.java [hunk 2]
```diff
@@ -43,7 +43,7 @@ public RCTAEPPlacesModule(ReactApplicationContext reactContext) {
 
   @Override
   public String getName() {
-    return "RCTAEPPlaces";
+    return "AEPPlaces";
```
Comment (yangyansong-adbe) [inline packages/places/android/src/main/java/com/adobe/marketing/mobile/reactnative/places/RCTAEPPlacesModule.java:46]: Good catch! @namArora3112

File: packages/places/src/Places.ts [hunk 3]
```diff
@@ -28,15 +28,34 @@ interface IPlaces {
   clear: () => void;
   setAuthorizationStatus: (authStatus?: PlacesAuthStatus) => void;
 }
+ 
+const logNativeModules = () => {
```
Comment (yangyansong-adbe) [inline packages/places/src/Places.ts:32]: Let's remove the test code before merging the pull request.

**Review by yangyansong-adbe — APPROVED** [review]
LGTM, only several small comments

File: apps/AEPSampleAppNewArchEnabled/app/PlacesView.tsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
+/*
+Copyright 2022 Adobe. All rights reserved.
```
Comment (cacheung) [inline apps/AEPSampleAppNewArchEnabled/app/PlacesView.tsx:2]: 2024

**Review by cacheung — COMMENTED** [review]
If we merge this PR to new_arc, and later to main, do we need to release Place right away to npm or we can have the Place source code updated in main, but release to npm later?
@yangyansong-adbe

**Comment by cacheung** [comment]
> If we merge this PR to new_arc, and later to main, do we need to release Place right away to npm or we can have the Place source code updated in main, but release to npm later?

Talked to @yangyansong-adbe , we can release Place along with new_arch release.

File: apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainApplication.kt (hunk 1 above)
Comment (namArora3112) [reply apps/AEPSampleAppNewArchEnabled/android/app/src/main/java/com/AEPSampleAppNewArchEnabled/MainApplication.kt:77]: removed

File: apps/AEPSampleAppNewArchEnabled/app/PlacesView.tsx (hunk 4 above)
Comment (namArora3112) [reply apps/AEPSampleAppNewArchEnabled/app/PlacesView.tsx:2]: done

File: packages/places/src/Places.ts (hunk 3 above)
Comment (namArora3112) [reply packages/places/src/Places.ts]: removed

File: packages/places/android/src/main/java/com/adobe/marketing/mobile/reactnative/places/RCTAEPPlacesModule.java (hunk 2 above)
Comment (namArora3112) [reply packages/places/android/src/main/java/com/adobe/marketing/mobile/reactnative/places/RCTAEPPlacesModule.java]: raising a separate PR for the same . isolating tsx and places fixes with different PR

**Comment by namArora3112** [comment]
> > If we merge this PR to new_arc, and later to main, do we need to release Place right away to npm or we can have the Place source code updated in main, but release to npm later?
> 
> Talked to @yangyansong-adbe , we can release Place along with new_arch release.

Hey I have isolated places fixes branch and adding tsx files only in this PR only.

File: apps/AEPSampleAppNewArchEnabled/android/app/src/main/AndroidManifest.xml [hunk 5]
```diff
@@ -4,6 +4,10 @@
   <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
   <uses-permission android:name="android.permission.VIBRATE"/>
   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+  <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
+<uses-permission android:name="android.permission.ACCESS_COARSE_LOCATION" />
```
Comment (cacheung) [inline apps/AEPSampleAppNewArchEnabled/android/app/src/main/AndroidManifest.xml:8]: nit: can we align these two with others?

**Review by cacheung — APPROVED** [review]
Just one minor comment.

File: apps/AEPSampleAppNewArchEnabled/android/app/src/main/AndroidManifest.xml (hunk 5 above)
Comment (namArora3112) [reply apps/AEPSampleAppNewArchEnabled/android/app/src/main/AndroidManifest.xml:8]: sure

**Pull request merged**